### PR TITLE
Add royal jelly in insect mutagen recipe

### PR DESCRIPTION
**Pull request by SariusSkelrets**
#### Summary
Features "Allow crafting insect mutagen with royal jelly"

#### Purpose of change

Royal jelly is both undoubtely insect-related and rarely used.  I thought of something to give it more uses

#### Describe the solution

Add royal jelly in the insect mutagen recipe

#### Describe alternatives you've considered

not doing it 

#### Testing

create a world, debug learn recipes, check insect mutagen

#### Additional context

I don't plan to add another version of the royal jelly for now

### CODE REVIEW

**Comment by anothersimulacrum** [comment]
*Features

**Comment by SariusSkelrets** [comment]
> *Features

Edited. Thanks!

File: data/json/recipes/chem/mutagens.json [hunk 1]
```diff
@@ -115,7 +115,7 @@
     "using": [ [ "mutagen_production_standard", 25 ] ],
     "components": [
       [ [ "mutagen", 1 ] ],
-      [ [ "honey_bottled", 3 ], [ "honeycomb", 3 ], [ "bee_sting", 3 ], [ "wasp_sting", 3 ] ],
+      [ [ "honey_bottled", 3 ], [ "honeycomb", 3 ], [ "royal_jelly", 1 ], [ "bee_sting", 3 ], [ "wasp_sting", 3 ] ],
```
Comment (Maleclypse) [inline data/json/recipes/chem/mutagens.json:118]:
```suggestion
      [ [ "honeycomb", 3 ], [ "royal_jelly", 1 ], [ "bee_sting", 3 ], [ "wasp_sting", 3 ] ],
```

**Review by Maleclypse — COMMENTED** [review]
Unless I'm reading this incorrectly these are all options.   I think removing bottled honey makes sense since it would have likely gone through several industrialized processes.  Or you could make it have several different mandatory component options.

**Comment by Venera3** [comment]
While you're here, would you include their eggs? Those at least have genetic material to work off of, but I haven't gotten around to adding them to mutagen crafting - currently ants, fireflies and dragonflies have eggs (and centipedes, but let's not stretch "insect" too far).

**Comment by Mom-Bun** [comment]
Does royal jelly contain insect DNA?

**Comment by SariusSkelrets** [comment]
I didn’t found anything about royal jelly DNA but the change isn’t only to add an use

The royal jelly is what makes the larvae become queens. It fits thematically in a recipe aiming to change you in a half-insect human


@Venera3 there’s no bee eggs in the game. I’ll add the eggs you suggested but adding bee eggs won’t be a part of this PR. 

To reduce ingredient clutter, I’ll try to add the insect template eggs as one all-including valid ingredient since there’s currently ant, spider, roach, locust, dragonfly, firefly and centipede eggs ingame

**Comment by kevingranade** [comment]
If you're already a bee, it might induce you to become a queen bee.
Since what you want to bee is a human/insect hybrid that dedicates something less than 90% of their energy to maintaining their reproductive organs, I respectfully suggest that you do not actually want to bee a queen hybrid.
If you did want a queen transformation in a mod since I don't think we're touching *that* with a km pole, it would probably treat royal jelly as (or an ingedient of) queen bee serum.

**Pull request closed without merging**